### ISSUE TITLE
Hi, Kishor10d

### ISSUE THREAD

**Issue by itsdiegodutra** [kishor10d/Hotel-Room-Booking-System#10]
Friend, I'm from Brazil, beginner in programming, I downloaded your source code and found it amazing. I would like to ask you if this project was continued? This version available here does not finalize the reservation, could you help me with this? Hugs!

**Comment by kishor10d** [comment]
@diegofdutra : Yes this project is maintained, but still it is not production ready.
The booking module is already developed, but it takes booking date-wise only and not hour-wise.
Also, check-in, check-out and cancel booking part is still under development.
